### DISCUSSION TITLE
Performance Issues while having OBS open (7900 XTX)

### DISCUSSION THREAD

**Discussion by Stellification** [ppy/osu#23078] [Q&A] [answered]
Since I upgraded from an nvidia 2060 super to a radeon 7900 XTX, playing osu! on stream became basically impossible.

hardware info: 

CPU: intel 13600-KF
GPU: radeon 7900 XTX
RAM: 32gb 6800mhz
Screen: 240hz
OS: Win10

When not streaming and simply having OBS open (or any streaming software for that matter), I am able to reach 12500 fps (0.08ms frametime). Just opening OBS reduces it to 600 fps (1.6ms frametime) while also having a lot of microstutters and varying refresh rates.

Changing the fps limit to "optimal" reduces performance further to 400-500 fps while keeping the same visual issues and enabling vsync caps it to 240, but nearly consistently erases the stuttering issues while also keeping constant 240 hz.

Playing with vsync on works, however theres very noticeable input delay when aiming, so this isnt an optimal fix.

Things I've already tried:

- Update gpu drivers or rolling them back
- Change OBS version
- osu lazer with a different encoder
- disabling windows game mode
- change various GPU settings in the amd adrenaline software

any advice and is this issue already known? couldnt find anything related to it unfortunately

**Comment by Cootz** [comment]
Have you tried using vulkan render?

**Reply by Axilotl17** [reply]
disabled in current ver

**Comment by Theighlin** [comment]
This doesn't sound like lazer? Fps should cap at 1000 and there's no optimal limiter setting. Stable issues don't belong here. Also this is very abstract, so i don't think this should even be reported to the stable issues repo. I think your best bet is ask on the osu forums.

**Reply by Stellification** [reply]
this applies to stable and lazer, as well as direct3d and opengl renderer in lazer

**Comment by peppy** [comment, marked as answer]
You should not be able to run above 1000 Hz. Queries like this is one major reason we added this limit (gross misunderstanding of how high an optimal FPS sending to a GOU should be).

https://osu.ppy.sh/help/wiki/Performance_Troubleshooting may be of help to you to read but we can't support your use case here.

**Reply by Stellification** [reply]
even when running on optimal (which on my 2060 super put it to 800-ish fps) it creates very weird refresh rate changes and cant hold stable fps at all.

i dont want to run the game above 1000hz, it was just an example of how severe the difference is when running OBS in the background

**Comment by aizuon** [comment]
The issue can be resolved by disabling ShadowPlay completely

![image](https://github.com/ppy/osu/assets/44760523/5c1d6748-379a-4d5a-a62c-7abe3de562a0)

Such a weird occurrence
